### DISCUSSION TITLE
Please Update ModifyStageVar to Include All Parameters Under the Info Block

### DISCUSSION THREAD

**Discussion by MarioManX1983** [ikemen-engine/Ikemen-GO#2211] [Feature Request] [answered]
### Is your feature request related to a problem? Please describe.

I'm trying to make a stage with helpers (through Attached Char) that only appear when a certain version of the stage is active. This could be easily doable if the StageVar trigger could read the Parameters under the [Info] Block. Specifically, the name or displayname Parameters.

### Describe the solution you'd like

The reason why I'm asking for the ModifyStageVar to be updated is because StageVar can read all the same Parameters as ModifyStageVar can.

Alternatively, there could be a new trigger called StageElemActive(ID) and !StageElemActive(ID). It would look at a stage's elements according to the specified ID and see if that element is active or not.

### Search terms

New, new, Feature, feature, Stage, stage, Trigger, trigger,

**Comment by potsmugen** [comment, marked as answer]
The name is accessible since Mugen, which is why it's not documented in our wiki.
https://www.elecbyte.com/mugendocs-11b1/trigger.html#stagevar

With that said there are some parameters StageVar and ModifyStageVar can't access yet because stages went through a lot of changes recently.
